webapp/store/app/saving/actions.js

```diff
@@ -0,0 +1,19 @@
+import Counter from '@core/counter'
+
+export const APP_SAVING_UPDATE = 'app/saving/update'
+const counter = new Counter()
```

Comment:
why do we need this counter?

Reply:
checking how many times the use clicks on saving